packages/itwin/tree-widget/src/tree-widget-react/components/trees/common/components/ProgressOverlay.tsx

```diff
@@ -4,13 +4,13 @@
  *--------------------------------------------------------------------------------------------*/
 
 import "./ProgressOverlay.css";
-import { Spinner } from "@itwin/itwinui-react/bricks";
+import { ProgressBar } from "@itwin/itwinui-react/bricks";
 
 // TODO: replace spinner with ProgressLinear equivalent
 export function ProgressOverlay() {
   return (
     <div className="tw-progress-overlay-container">
-      <Spinner />
+      <ProgressBar aria-labelledby={"tw-search-box-button"} />
```

Comment:
I don't think `ProgressBar` should be `labelledBy` search button. I think you should add `VisuallyHidden` with text that describes what progress bar is doing.

Reply:
adjusted